CleanWeather/Scenes/CityListMVVM/CityListControllerMVVM.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  CityListControllerMVVM.swift
+//  CleanWeather
+//
+//  Created by Marek Skrzelowski on 09/01/2020.
+//  Copyright © 2020 vandermesis. All rights reserved.
+//
+
+import UIKit
+
+final class CityListControllerMVVM: SharedViewController {
+
+    @IBOutlet private weak var cityListTableView: UITableView!
+
+    private var citiesWeatherDataSource = [CityWeather]()
```

Comment:
Keep the data source in view model that’s the whole idea of MVVM

Reply:
I didn't notice that it is not needed here. That var is not used anywhere. I just forgot to delete it after refactoring. :/

Reply:
But I get a point about not using data source in controller but only in viewmodel. :) I didn't know that.